src/identity-providers/add/AddMapper.tsx

```diff
@@ -372,92 +382,201 @@ export const AddMapper = () => {
+              <>
+                {" "}
+                <FormGroup
+                  label={t("common:attributes")}
+                  labelIcon={
+                    <HelpItem
+                      helpText="identity-providers-help:attributes"
+                      forLabel={t("attributes")}
+                      forID={t(`common:helpLabel`, { label: t("attributes") })}
+                    />
+                  }
+                  fieldId="kc-gui-order"
+                >
+                  <AttributesForm
+                    form={form}
+                    inConfig
+                    array={{ fields, append, remove }}
                   />
-                )}
-              />
-            </FormGroup>
-            <FormGroup
-              label={t("common:role")}
-              labelIcon={
-                <HelpItem
-                  id="name-help-icon"
-                  helpText="identity-providers-help:role"
-                  forLabel={t("identity-providers-help:role")}
-                  forID={t(`identity-providers:helpLabel`, {
-                    label: t("role"),
-                  })}
-                />
-              }
-              fieldId="kc-role"
-              validated={
-                errors.config?.role
-                  ? ValidatedOptions.error
-                  : ValidatedOptions.default
-              }
-              helperTextInvalid={t("common:required")}
-            >
-              <TextInput
-                ref={register()}
-                type="text"
-                id="kc-role"
-                data-testid="mapper-role-input"
-                name="config.role"
-                value={selectedRole[0]?.name}
+                </FormGroup>
+                <FormGroup
+                  label={t("regexAttributeValues")}
+                  labelIcon={
+                    <HelpItem
+                      helpText="identity-providers-help:regexAttributeValues"
+                      forLabel={t("regexAttributeValues")}
+                      forID={t(`common:helpLabel`, {
+                        label: t("regexAttributeValues"),
+                      })}
+                    />
+                  }
+                  fieldId="regexAttributeValues"
+                >
+                  <Controller
+                    name="config.are-attribute-values-regex"
+                    control={control}
+                    defaultValue="false"
+                    render={({ onChange, value }) => (
+                      <Switch
+                        id="regexAttributeValues"
+                        data-testid="regex-attribute-values-switch"
+                        label={t("common:on")}
+                        labelOff={t("common:off")}
+                        isChecked={value === "true"}
+                        onChange={(value) => onChange("" + value)}
+                      />
+                    )}
+                  />
+                </FormGroup>{" "}
```

Comment:
Perhaps this white space can be removed?
```suggestion
                </FormGroup>
```